spatialos.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "unity_gdk",
     "project_version": "0.0.1",
-    "sdk_version": "14.0-b6841-3d544-WORKER-SNAPSHOT",
+    "sdk_version": "13.5.1-b8033-b6bcd-WORKER-SNAPSHOT",
```

Comment:
Do we want to use `13.5.1-gdk-for-unity` here?

Reply:
Yes - please make this change